### DISCUSSION TITLE
Nuxt 2 + community maintenance

### DISCUSSION THREAD

**Discussion by dargmuesli** [nuxt/nuxt#17655] [Questions] [answered]
Hey @pi0 and @danielroe,
thank you very much for your continued efforts with the nuxt project!

I am wondering about the [expected roadmap date for `nuxt@2.16`](https://v3.nuxtjs.org/community/roadmap/) (April 2022) and about maintenance for `nuxt-community` modules. There are e.g. a few open PRs to add bridge support for community modules, but seemingly there are only very few reviews. It feels like the Nuxt team may profit from an additional team member who can take care of those matters so that you can continue your focus on Nuxt 3. Would you be open to that? What are your thoughts?

I suppose that would be a welcomed insight for your community. Thanks once more for your work!

**Comment by pi0** [comment, marked as answer]
Hi @dargmuesli and thanks for volunteering to help and kind words ❤️ Indeed we lowered focus on Nuxt 2 for the moment to stabilize Nuxt 3 as soon as possible and then backport as much as possible to Nuxt 2 via bridge. Your help is more than welcome for community modules that have not active maintainer and need support for Nuxt 3. As well as [nuxt/bridge](https://github.com/nuxt/bridge) which needs issue triage and backporting the latest improvements. Feel free to message me in discord (pi0#6582) would be happy to chat more and find any opportunities you can be involved.
